src/commands/deploy/deploy.mjs

```diff
@@ -178,31 +178,6 @@ const validateFolders = async ({ deployFolder, functionsFolder }) => {
   return { deployFolderStat, functionsFolderStat }
 }
 
-const getDeployFilesFilter = ({ deployFolder, site }) => {
```

Comment:
Unlike `folder-walker`, `fast-glob` does not accept a file filter function.

Therefore this logic was adapted for the `fast-glob` file queries that exist in `src/utils/deploy/hash-files.mjs`.